modules/DNN/Neuron.h

```diff
@@ -1,14 +1,54 @@
 #pragma once
 
+//#include "DGM.h"
+//#include "DNN.h"
 #include "types.h"
+#include <random>
 
-namespace DirectGraphicalModels { namespace dnn
-{
-	class CNeuron
-	{
-	public:
-		DllExport CNeuron(void) { printf("CNeuron constructor\n"); }
-		DllExport ~CNeuron(void) = default;
-	private:
-	};
-} }
\ No newline at end of file
+
+namespace DirectGraphicalModels {
+    namespace dnn
+    {
+        class CNeuron
+        {
+        public:
+            static const int SIZE= 76;
+
+            DllExport CNeuron(void) {
+                //printf("CNeuron constructor\n");
+            }
+            DllExport ~CNeuron(void) = default;
+            
+            void setNodeValue(double thisValue) {
+                m_value = thisValue;
+            }
+
+            double getNodeValue() {
```

Comment:
this method should be const, because getters should not modify the class:
`double getNodeValue() const`

Reply:
done